### PR TITLE
#3731 post fixes

### LINKED ISSUE #3731: Post white-on-white and black-on-black fixes
Seems #3730 post-fixes 👀
 
Make readable white texts on a white background by making them black?

- [ ] Creating on-chain identity needs better readability as well, currently there's no difference between the text which was already filled out (Handle) and text which is serving as a place holder
image

- [ ] collection select dropdown:
image

- [ ] Search on black version, white on white text
image

- [ ] On landing?
image

- [x] Action - Price
image

- [x] Carousel?
image

- [x] Black on Black? 😅
image

- [x] Sharing on collection
image

Probably there will be other white-on-white text? 
Keep looking for bugs 👀 

### Ref 
- #3730 
- #3713

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #3731
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by roiLeo** [comment]
I can't see no difference in `gallery-search` icon could you please share screenshot?

+ find another issue in darkmode, I can't see what I'm typing 
Screenshot 2022-08-19 at 07-59-54 KodaDot - Kusama NFT Market Explorer

**Comment by yangwao** [comment]
> * I can't see what I'm typing

Yeah, I thought I reported it somewhere; thanks for the reinforcement 😅

**Comment by prachi00** [comment]
> gallery-search
Screen Shot 2022-08-20 at 4 50 04 PM


removing the gallery-search class will make the icon white

**Comment by prachi00** [comment]
> I can't see no difference in `gallery-search` icon could you please share screenshot?
> 
> * find another issue in darkmode, I can't see what I'm typing
>   Screenshot 2022-08-19 at 07-59-54 KodaDot - Kusama NFT Market Explorer

fixed

File: components/Navbar.vue [hunk 1]
```diff
@@ -399,8 +399,12 @@ export default class NavbarMenu extends mixins(PrefixMixin) {
...
     min-width: 140px;
     margin: 0 1rem;
-    .icon {
-      color: $placeholder-color !important;
+    .gallery-search {
+      .control {
+        .icon {
+          color: $placeholder-color !important;
+        }
+      }
```
Comment (roiLeo) [inline components/Navbar.vue:407]: I don't understand why you need to rewrite this rule?

**on beta already work**
Capture d’écran 2022-08-22 à 9 48 03 AM

**on preview** (rewriting rule)
Capture d’écran 2022-08-22 à 9 48 16 AM

personnal preference: no need to write multiple line only add `.gallery-search .control .icon`

File: styles/global.scss [hunk 2]
```diff
@@ -5,7 +5,10 @@ body {
...
   .select select option,
   .taginput .taginput-container.is-focusable,
-  .textarea,
+  .textarea {
+    color: $select-color;
+  }
+
   .input {
     color: $input-color;
   }
```
Comment (roiLeo) [inline styles/global.scss:14]: why different color between input & textarea?

**Comment by yangwao** [comment]
Is everything good to go? :)

**Review by roiLeo — APPROVED** [review]
lgtm

**Comment by yangwao** [comment]
pay 50

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 49.71 USD/KSM ~ 1.006 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x1766a7e911d600af7b7029dc1a08b7a323fbcba722ca8ae1adb7c0326d68f432](https://kusama.subscan.io/extrinsic/0x1766a7e911d600af7b7029dc1a08b7a323fbcba722ca8ae1adb7c0326d68f432)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**